docs/books/incus_server/10-automating.md

```diff
@@ -61,7 +61,7 @@ Your entry will look like this:
 
 Save your changes and exit.
 
-This will create a log in incusadmin's home directory called "refreshlog" which will give you knowledge of whether your process worked or not. Very important!
+This will create a log-in incusadmin's home directory called "refreshlog," which will give you knowledge of whether your process worked or not. Very important!
```

Comment:
```suggestion
This will create a log in incusadmin's home directory called `refreshlog`, which will give you knowledge of whether your process worked or not. Very important!
```